lap/settings.py

```diff
@@ -74,8 +75,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
Let's just leave sqlite in the repo, so we don't have DB usernames / passwords in the repo (even if they are placeholders).